draft.xml

```diff
@@ -191,6 +191,144 @@ Content-Type: application/json
       </section>
     </section>
     
+    <section title="Authenticator Association List Endpoint">
+      <t>
+        The authenticator list endpoint is an OAuth 2.0 endpoint defined in
+        this document that is designed to allow listing all the authenticators
+        associated with a resource owner.  The authenticator list endpoint
+        MUST accept HTTP GET messages.  The authenticator
+        list endpoint MUST be protected by a transport-layer security
+        mechanism.
+      </t>
+      <t>
+        The authenticator associate endpoint MUST be an OAuth 2.0 protected
+        resource and it MUST accept access tokens as defined by <xref target='RFC6750'/>
+        to limit listing of authenticators to only previously authorized parties.  The method
+        by which the access token is obtained by the client is generally out of
+        band; one of such mechanisms is defined by this specification.  The method
+        by which the access token is verified and validated by the authenticator
+        associate endpoint is out of scope for this specification.
+      </t>
+        
+      <section title='Authenticator Association List Request' anchor="list-req">
+        <t>
+          The client makes a request to the authenticator list endpoint
+          including no parameters.
+        </t>
+        <t>
+          The access token MUST be sent as a Bearer Token, per section 2 of
+          <xref target='RFC6750'/>.  It is RECOMMENDED that the access token be
+          sent using the Authorization header field.
+        </t>
+        <figure>
+          <preamble>
+            For example, the client makes the following HTTPS request (line
+            breaks are for display purposes only)::
+          </preamble>
+          <artwork><![CDATA[
+GET /associate HTTP/1.1
```

Comment:
To be sincere I don't like the url, in particular this one, using GET over a verb as part of an API does not look clear to me, except that I were rendering something, but if I change it gets inconsistent with the DELETE method, I would user 

GET /authenticators and DELETE /authenticators/{id} but then the rational about mimic https://tools.ietf.org/html/rfc7592#section-2.3 gets broken, and /associate endpoint gets
quite inconsistent with the rest. 

What do you think?

What about
POST /mfa/associations instead of POST /mfa/associate
GET /mfa/associations instead of GET /mfa/associate
DELETE /mfa/associations/{id} instead of DELETE /mfa/associate/{id}

Or even
POST /mfa/authenticators instead of POST /mfa/associate
GET /mfa/authenticators instead of GET /mfa/associate
DELETE /mfa/authenticators/{id} instead of DELETE /mfa/associate/{id}

I know that auth concepts don't usually match REST well but I think in this case it kind of does